devices/MulticastDNS/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Iot.Device.Multicast")]
```

Comment:
For consistency, I would suggest:

```suggestion
[assembly: AssemblyTitle("Iot.Device.MulticastDns")]
```